### ISSUE TITLE
How BatchNorm is handled?

### ISSUE THREAD

**Issue by liqimai** [lucidrains/ema-pytorch#29]
How does ema_pytorch handle BatchNorm statistics? Are BatchNorm statistics also averaged over steps? Should they be averaged?
